packages/webpack-config/src/webpack/base/settings/output/legacy.js

```diff
@@ -5,6 +5,6 @@ export const legacyOutputSettings = {
     path: appTarget,
     publicPath,
     filename: "js/[name].legacy.js",
-    chunkFilename: "js/chunks/[name].legacy.[hash].js",
+    chunkFilename: "resources/chunks/[name].legacy.[hash].js",
```

Comment:
Does this really work, if my resources folder is not `resources`? Shouldn't this be `resourcesHome` from the config?

Reply:
this ist not an issue, to clarify i added a js folder to the path